packages/wrangler/e2e/dev.test.ts

```diff
@@ -106,6 +106,39 @@ describe.each([
 
 		await expect(fetchText(url)).resolves.toMatchSnapshot();
 	});
+
+	// regression test for https://github.com/cloudflare/workers-sdk/issues/5297
+	// The runtime inspector can send messages larger than 1MB limit websocket message allowed by workers.
+	// Connecting directly to the inspector works fine, but we proxy the inspector messages
+	// through a worker (InspectorProxyWorker) which hits the limit.
+	// By logging a large string (2MB) and verifying that wrangler logs it to the console,
+	// we can verify that the inspector messages are being proxied successfully.
+	it.only("can receive large messages from the inspector", async () => {
```

Comment:
```suggestion
	it("can receive large messages from the inspector", async () => {
```

Reply:
Draft PR for now because I want I still need to confirm this regression test fails without the fix

And I'm waiting for the workerd release before I can do that (without building workerd myself)